src/qualimap/qualimap_rnaseq/config.vsh.yaml

```diff
@@ -0,0 +1,111 @@
+name: "qualimap"
+info:
+  migration_info:
+    git_repo: https://github.com/nf-core/rnaseq.git
+    paths: [modules/nf-core/qualimap/rnaseq/main.nf]
+    last_sha: 54721c6946daf6d602d7069dc127deef9cbe6b33
+description: |
+  Qualimap RNA-seq QC RNA-seq QC reports quality control metrics and bias estimations 
+  which are specific for whole transcriptome sequencing, including reads genomic 
+  origin, junction analysis, transcript coverage and 5’-3’ bias computation.
```

Comment:
Could you fill in the missing metadata?

Also, the 'migration_info' can be omitted, because this component should be written from scratch